make/photon/log/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM vmware/rsyslog-photon:8.15.0
+FROM vmware/rsyslog-photon:8.15.0-6
```

Comment:
Shall we use stage build?
https://docs.docker.com/engine/userguide/eng-image/multistage-build/#name-your-build-stages

Reply:
For this special case we only need to copy some env files instead of build/compile something.

Not sure what benefit will we get from the stage build?